src/solver/gemm.cpp

```diff
@@ -87,6 +87,13 @@ bool GemmFwdBase::IsApplicable(const ExecutionContext& ctx,
     const auto& xDesc = problem.GetIn();
     const auto& wDesc = problem.GetWeights();
     const auto& yDesc = problem.GetOut();
+    if(xDesc.GetType() == miopenInt8x4 || xDesc.GetType() == miopenInt8)
+    {
+        // rocBlas needs the output to be int32 always
```

Comment:
[Q] A bugfix in Forward Gemm solvers? How important is it? Maybe it worth a dedicated PR?

Reply:
Potentially yes, however, lets keep it here to avoid churn.

Reply:
Okay. Can you please then update the name of the PR with smth like "Small fix in Fwd GEMM solvers"?

Reply:
Resolved.